index.js

```diff
@@ -1,11 +1,20 @@
 const express = require('express')
 const app = express()
 const Dictionary = require('./src/dictionary.js')
+const WordReader = require('./src/word-reader.js')
+const ConvertJson = require('./src/convert-object-json.js')
+const Game = require('./src/game.js')
+const MapDataString = require("./assets/saved-games.json");
+let MapDataGames =new Map();
+
 Dictionary.getWord().then(word => console.log('word: ', word))
 
-app.get('/game', (req, res) => {
-    res.send({
-        id: 1,
+MapDataGames = ConvertJson.objectToStrMap(MapDataString);
+
+app.get('/gameInstance', (req, res) => {

```

Comment:
what is the purpose of this endpoint? the task for today was the GET /game only...